WordPress/src/main/java/org/wordpress/android/ui/accounts/AbstractFragment.java

```diff
@@ -301,6 +309,10 @@ public void onErrorResponse(VolleyError error) {
             } else {
                 showError(messageId);
             }
+
+            if (mStep == Step.CREATE_USER) {
+                AnalyticsTracker.track(AnalyticsTracker.Stat.CREATE_ACCOUNT_FAILED);
```

Comment:
Hey @daniloercoli , good point. I actually looked into adding error info to that track prior to opening the PR but it seems that the useful error message can be deeper in the response's body. Most of `onErrorResponse()`'s code is about extracting the message after all. It seemed too complex for the purposes of this PR and decided to leave it out for a future PR if we see the need.

That said, let me know if you think this is a blocker and we should address it within this PR. Thanks!